packages/semantic-ui-theme/semantic/src/themes/sharegate/elements/label.overrides

```diff
@@ -24,45 +27,37 @@
     fill: @marine500;
 }
 
-/* CONTENT | With Icon | LAYOUT | Colour | Naked*/
+/* CONTENT | With Icon | LAYOUT | Colour | Naked */
 .ui.label.with-icon.naked .icon {
     fill: initial;
 }
 
-/* CONTENT | With Button | LAYOUT */
-.ui.label.with-button,
-.ui.tag.label.with-button {
-    padding-right: 0.5rem;
-}
-
-/* CONTENT | With Icon | LAYOUT | Position */
-.ui.label.with-icon.with-icon-right {
-    padding-right: 0.5rem;
-}
-
 /* CONTENT | With Tag | LAYOUT */
 .ui.label.with-tag .tag {
     margin-right: @scaleBravo;
 }
 
-
-.ui.label.with-icon:not(.with-icon-right) {
-    padding-left: 0.5rem;
-}
-
+/* CONTENT | With Icon Right | LAYOUT */
 .ui.label.with-icon.with-icon-right .icon {
     margin-left: 0.25rem;
 }
 
+/* CONTENT | With Icon Left | LAYOUT */
 .ui.label.with-icon:not(.with-icon-right) .icon {
     margin-right: 0.25rem;
 }
 
+/* CONTENT | With Icon - Without text | LAYOUT */
 .ui.label.with-icon.without-text .icon {
     margin-right: 0;
 }
 
-/* mini */
+/* STATE | With Tag | Disabled */
+.ui.button.disabled .empty.circular.label {
+    opacity: 0.4;
+}
```

Comment:
I don't think this is how we should do it. A tag shouldn't support disabled only if he is embedded in a button.

The `<Tag />` component should accept a `disabled` prop which would then render the appropriate CSS class to add `opacity: 0.4`.

What do you think?

Reply:
Makes sense.